test/unittest/tests/block/test_extent_list_insert_wo_block.cpp

```diff
@@ -0,0 +1,167 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * block_ext.c: [extent_list2] Test extent list functions part 2. (More to come.)
+ *
+ * Test insert functions without block: __block_ext_insert, and __block_off_insert.
+ */
+
+#include <algorithm>
+#include <memory>
+
+#include <catch2/catch.hpp>
+
+#include "test_util.h"
+#include "../utils.h"
+#include "../utils_extlist.h"
+#include "../wrappers/mock_session.h"
+#include "wt_internal.h"
+
+using namespace utils;
+
+/*!
+ * To sort off_size by _off and _size
+ */
+struct {
+    bool
+    operator()(const off_size &left, const off_size &right)
+    {
+        return (left < right);
+    }
+} off_size_Less_off_and_size;
+
+TEST_CASE("Extent Lists: block_ext_insert", "[extent_list2]")
+{
+    /* Build Mock session, this will automatically create a mock connection. */
+    std::shared_ptr<MockSession> mock_session = MockSession::buildTestMockSession();
+    WT_SESSION_IMPL *session = mock_session->getWtSessionImpl();
+
+    std::vector<WT_EXT **> stack(WT_SKIP_MAXDEPTH, nullptr);
+
+    SECTION("insert into an empty list has one element")
+    {
+        BREAK;
+        /* Setup */
```

Comment:
WT comment style requires a full stop at the end of every comment.

Reply:
Done.